src/axom/sidre/docs/sphinx/parallel_io_concepts.rst

```diff
@@ -178,14 +180,14 @@ knowledge.
 User-specified data in the root file
 ------------------------------------
 
-The root file is automatically created to provide the IOManager with
-bookkeeping information that is used when reading data, but it can also
+The root file is automatically created to provide the ``IOManager`` object 
+with bookkeeping information that is used when reading data, but it can also
 be used to store additional data that may be useful to the calling code or
 is needed to allow other tools to interact with the data in the output files,
-such as for visualization.  For example, Conduit's blueprint index can be
-:ref:`stored in a DataGroup <sidre-conduit>` written to the root file
-to provide metadata about the
-mesh layout and data fields that can be visualized from the output files.
+such as for visualization.  For example, a ``Conduit Mesh Blueprint <https://llnl-conduit.readthedocs.io/en/latest/blueprint.html>`_ index can be 
```

Comment:
This link is broken. I think the following will work:

```suggestion
such as for visualization.  For example, a `Conduit Mesh Blueprint <https://llnl-conduit.readthedocs.io/en/latest/blueprint.html>`_ index can be 
```

Reply:
Fixed. extra '`' at the beginning.